dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers/get-keying-sheet-for-report-id.ts

```diff
@@ -0,0 +1,68 @@
+import { In } from 'typeorm';
+import { FeeRecordPaymentJoinTableEntity, FeeRecordStatus } from '@ukef/dtfs2-common';
+import { SqlDbDataSource } from '@ukef/dtfs2-common/sql-db-connection';
+import { mapFeeRecordEntityToKeyingSheetRowWithoutFeePayments } from '../../../../../mapping/keying-sheet-mapping';
+import { KeyingSheet, KeyingSheetFeePayment, KeyingSheetRow } from '../../../../../types/fee-records';
+
+/**
+ * Gets all the keying sheet fee record payment join table entries
+ * linked to the report with id matching the supplied report id
+ * @param reportId - The report id
+ * @returns The join table entries linked to the report id
+ */
+const getAndValidateKeyingSheetFeeRecordPaymentJoinTableEntries = async (reportId: number): Promise<FeeRecordPaymentJoinTableEntity[]> => {
```

Comment:
is the `andValidate` redundant here?

Reply:
There was originally some validation around this, but that was removed without updating the name 🙃